### PR TITLE
Mitigate Radio and Radio Option Formation Problems

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://1567-mitigate-radio-option--60f9b557105290003b387cd5.chromatic.com

## Description
Closes #1566
Closes #1567

## Testing done
Yep

## Screenshots
Screenshot 2023-02-24 at 5 01 10 PM
Screenshot 2023-02-24 at 5 06 05 PM

### CODE REVIEW

File: packages/web-components/src/global/_variables.scss [hunk 1]
```diff
@@ -32,13 +32,15 @@ $color-gray-warm-dark: #494440;
...
-$font-family-sans: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;
+$v3-color-base-darkest: #1b1b1b;
+$font-family-sans: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial,
+  sans-serif;
 $font-family-serif: Bitter, Georgia, Cambria, 'Times New Roman', Times, serif;
 $font-weight-normal: 400;
 $font-weight-bold: 700;
 $font-style-normal: normal;
 $font-style-italic: italic;
+$base-v3-font-size: 16px;
```
Comment (jamigibbs) [inline packages/web-components/src/global/_variables.scss:43]: This file is generated dynamically by the css-library (I forgot about this in previous reviews) so it has been updated in this PR already:

https://github.com/department-of-veterans-affairs/component-library/pull/624#discussion_r1119376756

Can changes here and in `_functions.scss` be reverted in this PR? Those updates are in `main` already.

**Comment by jamigibbs** [comment]
Some `rem` values to check:

Screenshot 2023-02-28 at 1 42 53 PM

Screenshot 2023-02-28 at 1 45 48 PM

**Comment by jamigibbs** [comment]
Storybook story translations buttons need the inline font size set to `16px` like we did here: 

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/storybook/stories/va-text-input-uswds.stories.jsx#L118-L126

Screenshot 2023-02-28 at 1 47 37 PM

**Comment by Andrew565** [comment]
Thanks Jami, I'll get on these fixes ASAP!

File: packages/web-components/src/components/va-radio-option/va-radio-option.scss [hunk 2]
```diff
@@ -71,13 +72,13 @@
   margin-top: 5px;
 }
 
-:host(:not([uswds])[tile="true"]) label {
+:host(:not([uswds])[tile='true']) label {
```
Comment (Andrew565) [inline packages/web-components/src/components/va-radio-option/va-radio-option.scss:75]: Thought I fixed this...

File: packages/web-components/src/global/_variables.scss [hunk 3]
```diff
@@ -84,4 +84,4 @@ $units-neg-7: -5.6rem;
 $units-neg-8: -6.4rem;
 $units-neg-9: -7.2rem;
 $units-neg-10: -8rem;
-$units-neg-15: -12rem;
\ No newline at end of file
+$units-neg-15: -12rem;
```
Comment (Andrew565) [inline packages/web-components/src/global/_variables.scss:87]: please ignore. :(

**Comment by Andrew565** [comment]
@jamigibbs Ready for re-review. A couple formatting changes snuck back in, I can try and revert them again if it's important.

**Comment by Andrew565** [comment]
Nevermind, found another bug

**Review by jamigibbs — APPROVED** [review]
This looks good. Thanks! 👍 

My only other recommendation is to update the title of the PR to something that's easier to understand. Something like, "Add Formation overrides to Radio and Radio Option v3 variations" or something. The related ticket numbers probably shouldn't be there.

And if you want to immediately close the related tickets when this is merged, just add a link to the github issue directly after the word "closes" in the description. Closes is a [github keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

**Comment by Andrew565** [comment]
Thanks Jami!

**Pull request merged**